### PR TITLE
SDK - Components - Enable loading graph components

### PR DESCRIPTION
**Pull request by Ark-kun**
The graph components are now correctly loaded and instantiated (ContainerOps are created in the process).
Also added pre-configured `ComponentStore.default_store`.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2010)

### CODE REVIEW

File: sdk/python/kfp/components/_component_store.py [hunk 1]
```diff
@@ -91,3 +92,24 @@ def load_component(self, name, digest=None, tag=None):
                 return comp._load_component_from_yaml_or_zip_bytes(response.content, url, component_ref)
 
         raise RuntimeError('Component {} was not found. Tried the following locations:\n{}'.format(name, '\n'.join(tried_locations)))
+
+    def _load_component_from_ref(self, component_ref : ComponentReference) -> Callable:
```
Comment (ajchili) [inline sdk/python/kfp/components/_component_store.py]: Nitpick, extra whitespace between variable name and `:`

File: sdk/python/kfp/components/_components.py [hunk 2]
```diff
@@ -276,3 +279,47 @@ def component_default_to_func_default(component_default: str, is_optional: bool)
...
+
+def _resolve_graph_task(graph_task: TaskSpec, graph_component_spec: ComponentSpec) -> TaskSpec:
+    from ..components import ComponentStore
+    component_store = ComponentStore.default_store
+
+    graph = graph_component_spec.implementation.graph
+
+    outputs_of_tasks = {}
+    def resolve_argument(argument):
+        if isinstance(argument, (str, int, float, bool)):
```
Comment (ajchili) [inline sdk/python/kfp/components/_components.py]: Are we able to use the `PrimitiveTypes` type here?

Comment (Ark-kun) [reply sdk/python/kfp/components/_components.py]: Unfortunately, no. Python complains that generic types cannot be used in `isinstance`.

File: sdk/python/kfp/components/_component_store.py (hunk 1 above)
Comment (Ark-kun) [reply sdk/python/kfp/components/_component_store.py]: Fixed.

**Pull request merged**